Generator/Templates/Core/class.sbntxt

```diff
@@ -8,11 +8,11 @@ namespace {{ namespace }}
     [Wrapper("{{ type_name }}")]
     public partial class {{ name }} : {{ parent }}
     {
-        internal new static GObject.Type GetGType() => new GObject.Type(Sys.{{ name }}.get_type());
+        protected internal new static GObject.Type GetGType() => new GObject.Type(Sys.{{ name }}.get_type());
         
         #region Constructors
-        protected {{ name }}(IntPtr ptr) : base(ptr) {}
-        protected {{ name }}(ConstructProp[] properties) : base(properties) {}
+        protected internal {{ name }}(IntPtr ptr) : base(ptr) {}
+        protected internal {{ name }}(ConstructProp[] properties) : base(properties) {}
```

Comment:
There is no `params` keyword for construct properties, is this intended ?

```suggestion
        protected internal {{ name }}(params ConstructProp[] properties) : base(properties) {}
```

Reply:
I don't think so.

@badcel?

Reply:
You are right, this was not intendet. Please add the params. Yesterday I was focused on getting it in a compileable state. I did not do any review. Thanks for having a critical lool @na2axl :+1: